### ISSUE TITLE
[Inbound Roaming][FEG] Create s8_proxy 

### ISSUE THREAD

**Issue by ssanadhya** [magma/magma#3595]
As discussed in the Inbound Roaming system architecture https://docs.google.com/document/d/1zJZpt5dCBjy54ODQ5DzbJZ2HufJgLxAIoVPyZPZL5UY/edit#heading=h.ysnq6v9qfm5k , we need an S8 proxy on FeG, which can:
- Communicate over GRPC with MME service on AGW
- Communicate over GTP-C to home PGW

**Comment by uri200** [comment]
PRs belonging to this issue

https://github.com/magma/magma/issues/3595
https://github.com/magma/magma/pull/3962
https://github.com/magma/magma/pull/4383
https://github.com/magma/magma/pull/4360
https://github.com/magma/magma/pull/4490
https://github.com/magma/magma/pull/5008
https://github.com/magma/magma/pull/5011
https://github.com/magma/magma/pull/5058
https://github.com/magma/magma/pull/5785

**Comment by uri200** [comment]
The tasks to be done to complete this issue are:
- Build a wrapper for Go-GTP - (done)
- Add `CreateSession` logic - 3 days (almost done)
- Add NH for S8_proxy logic - 3 days (testing may need to be done once terravm setup is complete)
- Setup and test create session loop with teravm - 1 day Still waiting for confirmation of the setup
- Add `ModifyBearer` logic - 5 days (coding + proto definition)
- Add `DeleteSession` logic - 5 days (coding + proto definition)
- Test `CreateSession+ModifyBerare+DeleteSession` on teravm - 3 days once setup is complete

Messages that will be created are:
```
CreateSessionRequest/Response
ModifyBerareRequest/Response
DeleteSessionRequest/Response
```

**Comment by uri200** [comment]
The basic features for s8_proxy in feg are already created. Please check the PRs included in my previous message. We have a service that has been tested against teravm. But hasn't been successfully tested yet against Hcore.

We are pending for a last PR to be approved to consider this issue complete. But I will create a new issue for the next sprint to track bugs or changes required once we start testing with s8_task from mme.

Note that s8_proxy has been created independently and we may require changes once MME s8_taks is completed
